documentation/src/docs/asciidoc/user-guide/writing-tests.adoc

```diff
@@ -367,6 +367,40 @@ NOTE: As of JUnit Jupiter 5.4, it is also possible to use methods from JUnit 4's
 `AssumptionViolatedException` to signal that a test should be aborted instead of marked
 as a failure.
 
+[[writing-tests-exceptions]]
+=== Other Exceptions
+
+JUnit Jupiter provides robust support for handling test exceptions. This includes the built-in mechanisms for managing test failures due to exceptions, the role of exceptions in implementing assertions, and how to specifically assert non-throwing conditions in code.
+
+[[writing-tests-uncaught-exceptions]]
+==== Uncaught Exceptions and Test Failures
+
+In JUnit Jupiter, if an exception is thrown and not caught within the test method, the framework will mark the test as failed.
+
+[source,java,indent=0]
+----
+include::{testDir}/example/exception/UncaughtExceptionHandlingDemoTestCase.java[tags=user_guide]
+----
+
+In the above example, the `testThatFailsDueToUncaughtException` method throws a `RuntimeException`. Since the exception is not caught within the test method, JUnit Jupiter will automatically fail the test.
```

Comment:
Question to the maintainers: Should this also explicitly mention that a `throws` clause has no effect, to avoid any misunderstanding?
For example
> [...] It does not matter whether or not the test method has a `throws` clause which lists the exception (or a superclass of it).

Otherwise users might assume `throws SomeException` means that it is expected that the test fails with `SomeException`.

Reply:
Standard Java semantics apply here, but that's not always readily apparent to someone new to testing.

So, yes, it would be beneficial for some users to point that out.

Reply:
added a section describing the throws clause in junit exception context